contrib/completions/bash/kubectl

```diff
@@ -628,6 +628,7 @@ _kubectl_run()
     flags_completion=()
 
     flags+=("--attach")
+    flags+=("--command")
```

Comment:
Is there a reason why you didn't make this look like exec, as proposed here:
https://github.com/GoogleCloudPlatform/kubernetes/issues/5581#issuecomment-85006570